src/vsr/replica.zig

```diff
@@ -8173,6 +8419,13 @@ pub fn ReplicaType(
             ));
 
             self.commit_min = self.superblock.working.vsr_state.checkpoint.commit_min;
+            self.upgrade_release = null;
```

Comment:
Maybe _not_ null this out? My understanding is that, when we hit this case, the following code is essentially `noreturn`. So, if we _don't_ null out here, we'll trip assertions if `release_transition` is buggy and doesn't actually reset the entire state!